src/app/globals.css

```
@@ -917,6 +1008,273 @@ body.modal-open .card {
+a.ui-btn[data-variant="danger"] {
+  border-color: var(--negative);
+  color: var(--negative);
+  background: color-mix(in oklab, var(--negative) 12%, transparent);
+}
+button.ui-btn[aria-busy="true"]::after {
+  content: "";
+  width: 14px;
+  height: 14px;
+  margin-left: var(--space-2);
+  border-radius: 50%;
+  border: 2px solid currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)